lib/response-handlers/foods-response-handler.js

```diff
@@ -1,6 +1,7 @@
 const url = `https://protected-basin-11627.herokuapp.com/`
 
 const handleResponse = require('../response-handlers/initial-response-handler')
+const renderFoods = require('../fetch-requests/foods-fetch-requests').renderFoods
 
 const postNewFood = (name, calories) => {
```

Comment:
I think all of this looks great! I would (most likely) organize this a bit differently.  I would call the fetch request (post foods), from the event listener, like you're doing, but the post foods fetch would live in the fetch request file, and it would call to your render foods function, which lives in your response handler file.  If your fetch call fails, you don't want to render the values you're passing through the event listener.  You only want to render once a successful post has completed. :)